alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/validation/FieldValidationUtility.java

```diff
@@ -47,13 +47,13 @@
     }
 
     public List<AlertFieldStatus> validateConfig(Map<String, ConfigField> fieldKeyToConfigField, Collection<FieldModel> fieldModels) {
-        logger.debug("Begin validating fields in configuration field model.");
+        logger.trace("Begin validating fields in configuration field model.");
```

Comment:
Changed these logs to `trace` level because they have not been useful for debugging and create a lot of noise at `debug` level